extra/release_tool.py

```diff
@@ -1989,6 +2041,11 @@ def do_docker_compose_branches_from_follows(state):
         state, "integration", state["integration"]["following"]
     )
 
+    # For the Docker images, use M.N as the release branch
+    mender_release_minor = re.match(
+        r"^([0-9]+\.[0-9]+)\.[0-9]+.*$", state["version"]
+    ).group(1)
```

Comment:
Won't this be missing the `.x`? The `M.N` Docker tags are stable releases, the `M.N.x` tags are unstable rolling branches. Probably we won't use the former anymore, but I think we should still stick to the convention.

Reply:
I thought about this too. The thing is that we don't have a way now to publish the M.N.x tags of the different services (following Mender product versions).

Thinking about it now, maybe this is something that we should do? It should be fairy simple from the template to query release_tool for the integration version and publish the mender-N.M.x images. Unless you disagree I'll do this also (this was a TBD point in the JIRA ticket, actually)

Reply:
Note to self: the version here is missing the `mender-` prefix.

Reply:
Fixed here; but I have now a TODO PR in `mendertesting` for these Docker tags to be actually updated.